source/contracts/reporting/ReportingToken.sol

```diff
@@ -163,4 +163,13 @@ contract ReportingToken is DelegationTarget, Typed, Initializable, VariableSuppl
         require(index < market.getNumberOfOutcomes());
         return payoutNumerators[index];
     }
+
+    function isIndeterminate() public constant returns (bool) {
```

Comment:
Would be nice to see a unit test of this method, mainly because I don't trust `.length` in Solidity (because its availability differs based on storage/memory).

Reply:
Will do!